src/gflownet/utils/metrics.py

```diff
@@ -522,6 +522,22 @@ def inv_transform(self, arr):
         return self.scale * arr + self.loc
 
 
+def chunkedsim(thresh, fp, mode_fps, delta=16):
```

Comment:
This doesn't seem to be called anywhere, although I can see how this is useful in a notebook.
I'd suggest a more expressive name, like `all_are_tanimoto_similar_to`

Reply:
Sounds, good how about `all_are_tanimoto_different`?

Reply:
Also, this is mostly for speed, it's a fair bit faster on large mode_fps